extensions/vscode/src/api/resources/ContentRecords.ts

```diff
@@ -55,35 +64,41 @@ export class ContentRecords {
     targetName: string,
     accountName: string,
     configName: string = "default",
+    params?: { dir?: string },
   ) {
-    const params = {
+    const data = {
```

Comment:
Changes the name here and above from `params` to `data` to better reflect the naming Axios suspects and to avoid confusion when using query params.